### PR TITLE
fix(db): db prefix support

### PR DESCRIPTION
**Pull request by ohadsc**

### CODE REVIEW

**Comment by ohadsc** [comment]
https://jira.yotpo.com/browse/YO-2443

File: app/code/community/Yotpo/Yotpo/sql/yotpo_setup/mysql4-install-1.6.4.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+$installer = $this;
+/* @var $installer Mage_Core_Model_Resource_Setup */
+
+Mage::log('mysql4-install-1.6.4.php');
+
+$installer->startSetup();
+
+$prefix = (string) Mage::getConfig()->getTablePrefix();
+Mage::log("prefix - " . $prefix);
```
Comment (omryYotpo) [inline app/code/community/Yotpo/Yotpo/sql/yotpo_setup/mysql4-install-1.6.4.php]: pls remove log writes

File: app/code/community/Yotpo/Yotpo/sql/yotpo_setup/mysql4-install-1.6.4.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+$base_table = "yotpo_rich_snippets";
+$table_with_quotes = "`" . $base_table .  "`";
+
+if ($prefix != "") {
+    $base_table = $prefix . $base_table;
+    $table_with_quotes =  "`". $base_table . "`";
```
Comment (omryYotpo) [inline app/code/community/Yotpo/Yotpo/sql/yotpo_setup/mysql4-install-1.6.4.php]: i don't like the name $table_with_quotes so much...
consider changing just  to table_name (your call)

**Comment by omryYotpo** [comment]
Looks great! 
Good job!! ⭐

**Pull request merged**